x/ibc/core/spec/01_concepts.md

```diff
@@ -7,6 +7,69 @@ order: 1
 > NOTE: if you are not familiar with the IBC terminology and concepts, please read
 this [document](https://github.com/cosmos/ics/blob/master/ibc/1_IBC_TERMINOLOGY.md) as prerequisite reading.
 
+## Client Creation, Updates, and Upgrades
+
+IBC clients are on chain light clients. The light client is responsible for verifying
+counterparty state. A light client can be created by any user submitting a client
+identifier and a valid initial `ClientState` and `ConsensusState`. The client identifier
+must not already be used. Clients are given a client identifier prefixed store to
+store their associated client state and consensus states. Consensus states are 
+stored using their associated height. 
+
+Clients can be updated by any user submitting a valid `Header`. The client state callback
```

Comment:
I think it's important to mention that the update is not necessarily the latest height.